addons/project_hr_expense/data/project_hr_expense_demo.xml

```diff
@@ -1,11 +1,47 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data noupdate="1">
+        <!-- Expense Sheet -->
+        <record id="transportation_expense_sheet" model="hr.expense.sheet">
+            <field name="name">Transportation Expense</field>
+            <field name="employee_id" ref="hr.employee_al"/>
+            <field name="state">approve</field>
+        </record>
+        <record id="restaurant_expense_sheet" model="hr.expense.sheet">
+            <field name="name">Restaurant Expense</field>
+            <field name="employee_id" ref="hr.employee_al"/>
+            <field name="state">approve</field>
+        </record>
+
         <record id="hr_expense.travel_admin_by_car_expense" model="hr.expense">
             <field name="analytic_distribution" eval="{ref('project.analytic_office_design'): 100}"/>
         </record>
         <record id="hr_expense.travel_demo_by_car_expense" model="hr.expense">
             <field name="analytic_distribution" eval="{ref('project.analytic_office_design'): 100}"/>
         </record>
+        <record id="transportation_expense" model="hr.expense">
+            <field name="name">Transportation</field>
+            <field name="employee_id" ref="hr.employee_admin"/>
+            <field name="analytic_distribution" eval="{ref('project.analytic_construction'): 100}"/>
+            <field name="product_id" ref="hr_expense.expense_product_meal"/>
+            <field eval="700.0" name="unit_amount"/>
+            <field name="product_uom_id" ref="uom.product_uom_unit"/>
+            <field eval="4.0" name="quantity"/>
+            <field eval="289.0" name="total_amount"/>
```

Comment:
```suggestion
            <field eval="700.0" name="unit_amount"/>
            <field name="product_uom_id" ref="uom.product_uom_unit"/>
            <field eval="4.0" name="quantity"/>
            <field eval="289.0" name="total_amount"/>
```
Shouldn't `total_amount` = `quantity` * `unit_amount` = 4 * 700 = 2800 +/- taxes ?
But no need to set the total amount as it will be computed, won't it?

Reply:
You are right. :slightly_smiling_face: